examples/mnist.py

```diff
@@ -24,7 +24,7 @@
 from blocks.main_loop import MainLoop
 
 
-def main(save_to, num_epochs):
+def main(save_to, num_epochs, no_bokeh):
```

Comment:
You should probably give a default, `no_bokeh=True` here as well, but make sure that the tests use `no_bokeh=True` so that it still gets tested on Travis.